### ISSUE TITLE
Bug: Invalid json output for invocations when running in dbt cloud ide

### ISSUE THREAD

**Issue by mbrummerstedt** [brooklyn-data/dbt_artifacts#259]
The insert statement for `database.schema.invocations` fails due to a single backslash instead of a double backslash or none in BigQuery.

The same issue was previously mentioned [here](https://github.com/brooklyn-data/dbt_artifacts/pull/242#issuecomment-1306209151) but was never fixed.

This happens if I run dbt_artifacts from the dbt cloud IDE, but it works fine, when I run dbt_artifacts in a job in production.

**Compiled code:**

```
/* {"app": "dbt", "dbt_version": "1.3.1", "profile_name": "user", "target_name": "dev", "connection_name": "master"} */

    
        insert into `development.dbt_dev_tech.invocations`
        VALUES
        
    
        (
        '4ca5e501-540c-485b-a69d-d7b498bde191', 
        '1.3.1', 
        'overall_project', 
        '2022-12-22 15:23:39.256605+00:00', 
        'build', 
        False, 
        'user', 
        'dev', 
        'dbt_dev', 
        16, 

        '', 
        '', 
        '', 
        '', 
        '', 

        
            null, 
        

        
            
            
                
            
            parse_json('{"use_probalistic_user_stitching_for_attribution": true}'), 
        

        parse_json('{"write_json": true, "use_colors": true, "printer_width": 80, "version_check": true, "partial_parse": true, "static_parser": true, "profiles_dir": "/usr/src/develop/.dbt", "send_anonymous_usage_stats": true, "event_buffer_size": 100000, "quiet": false, "no_print": false, "vars": "{}\n", "indirect_selection": "eager", "resource_types": [], "select": ["stg_orders_base__mysqldb_order_item_shipment_tracking"], "which": "build", "rpc_method": "build"}'), 
        parse_json('{}') 

        )
    
```

The issue that is causing issues is this part: `"vars": "{}\n"`

If it is changed to `"vars": "{}"` or `"vars": "{}\\n"` then it works as expected

**Comment by glsdown** [comment]
Hi @mbrummerstedt . Really sorry for the time it has taken to get back to you. Thank you for raising this issue.

I believe that the recently merged PR #285 should address this issue for you. I am aiming to get a release out this week which will include the change, so hopefully this will resolve the issue. Thanks again for raising.

**Comment by glsdown** [comment]
HI @mbrummerstedt . This issue fix is now available in v2.3.0 so I'm going to mark the issue as closed. Please feel free to open a new issue though if you are still having the same problem.
